### PR TITLE
Project README + cleanup

### PR DESCRIPTION
**Pull request by Cell00phane**

### CODE REVIEW

**Review by AlessioVerardo — APPROVED** [review]
Nice, we almost finish this project!!!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+
+## Going to an event with PolyEvents
+As a user attending an event, you will make the most out of your event with PolyEvents.
+
+You can view which events are happening, the way you like it. Whether as a simple list:
+
+<img src="docs/screenshots/events_list.png" alt="events list image" width="300"/>
+
+
+a timetable:
```
Comment (Mathematou) [inline README.md:17]: Nice, thank you for doing the readme

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/view/activity/EventActivityTest.kt [hunk 2]
```diff
@@ -48,7 +48,8 @@ import java.time.LocalDateTime
 import kotlin.test.*
 import org.mockito.Mockito.`when` as When
 
-
+// DISCLAIMER: These tests might fail due to a process crash on Cirrus. Consider thread sleeps in the
```
Comment (Mathematou) [inline app/src/androidTest/java/com/github/sdpteam15/polyevents/view/activity/EventActivityTest.kt:51]: Sad but true

**Review by Mathematou — APPROVED** [review]
Thank you for doing the readme and the cleanup

**Pull request merged**